pkg/router/f5/f5.go

```diff
@@ -1823,3 +1865,9 @@ func (f5 *f5LTM) deleteCertParts(routename string,
 
 	return nil
 }
+
+// escapeiControlUriPath returns a fully qualified `escaped` URI path that
+// can be used in the iControl REST calls.
+func escapeiControlUriPath(pathName string) string {
+	return strings.Replace(pathName, "/", "~", -1)
+}
```

Comment:
Why not put `escapeiControlUriPath` up with `iControlResourceUriPath` and `iControlVserverUriPath`?